web/cobrands/rutland/layout.scss

```diff
@@ -13,3 +13,12 @@ body.frontpage #site-logo {
     height: 100px;
     width: 220px;
 }
+
+
+body.fullwidthpage, body.twothirdswidthpage {
+    .container .content footer {
```

Comment:
Ugh, have opened https://github.com/mysociety/fixmystreet/issues/2107 to deal with this mess!

Reply:
(Also why is the logo layout only, not in base? Not something for here, but can't see why not.)